backend/python/app/models/enum.py

```diff
@@ -1,3 +1,4 @@
 from . import db
 
 enum = db.Enum("A", "B", "C", "D", name="enum")
+simple_entity_enum = db.Enum("A", "B", "C", "D", name="simple_entity_enum")
```

Comment:
What's the issue with using `enum` for the simple entity model?

Reply:
I got an error with the migration saying that the type "enum" already exists